tools/etcd-dump-logs/main.go

```diff
@@ -31,6 +35,8 @@ import (
 	"github.com/coreos/etcd/wal/walpb"
 
 	"go.uber.org/zap"
+	"bytes"
```

Comment:
Can you move `"bytes"` and `"os"` at the same level as `"encoding/hex"`? Just minor `gofmt` issue.